### PR TITLE
feat: add collectionId field to assetJson of sdk projects

### PR DESCRIPTION
**Pull request by leanmendoza**

### CODE REVIEW

File: src/sdk/project/asset-json.ts [hunk 1]
```diff
@@ -53,7 +55,11 @@ export namespace AssetJson {
       },
       category: WearableCategory.schema,
       rarity: Rarity.schema,
-      bodyShape: AssetWearableGender.schema
+      bodyShape: AssetWearableGender.schema,
+      collectionId: {
+        description: 'The collection where the wearable is placed.',
+        type: 'string'
```
[the remark this answers is not in the document]
Comment (leanmendoza) [reply src/sdk/project/asset-json.ts:61]: isn't it enough not to be in the required fields? or do you mean when you want to clean the collection id so the item stays at top root collection

[the remark this answers is not in the document]
Comment (menduz) [reply src/sdk/project/asset-json.ts:61]: Oh ok, that sounds compliant enough.

Comment (nicosantangelo) [inline src/sdk/project/asset-json.ts:61]: would it be useful to add a `format: 'uuid'` to this type?

Comment (leanmendoza) [reply src/sdk/project/asset-json.ts:61]: yep

File: src/sdk/project/asset-json.ts [hunk 2]
```diff
@@ -22,6 +22,8 @@ export type AssetJson = {
   category: WearableCategory
   rarity: Rarity
   bodyShape: AssetWearableGender
+
+  collectionId: string
```
Comment (gonpombo8) [inline src/sdk/project/asset-json.ts:26]: this key its not required at the schema validation, so shouldn't be optional ?

**Review by gonpombo8 — APPROVED** [review]

**Pull request closed without merging**